### DISCUSSION TITLE
Switch weight and trophy score around in the catch book entries

### DISCUSSION THREAD

**Discussion by Errandir-of-Gondolin** [janstehno/cotw-companion#98] [Ideas]
Small QoL improvement (at least for me): from top to bottom, the app currently first asks the trophy score, then the weight, then the trophy rating. But visually in the game the weight is at the top left of the harvest screen, with the trophy score right above the trophy rating at the bottom left of the harvest screen.

It would make it easier for my OCD brain to first have to complete the weight, then have my eyes scroll lower across the harvest screen to locate the trophy score and rating, rather than having to scan the screen up and down a few times to find all the info.

PS: saw the notification about the delayed app updates. All the best at uni, good luck with the exams (if you have any in January)! Won't deny that I'll be happy to see an update, since there are so many good ideas out there, but totally understandable that real life comes first.

**Comment by janstehno** [comment]
Hello, thank you for the support. I really appreciate it. The idea seems reasonable and the swap is really easy to do, so there should be no problem with this.

**Comment by janstehno** [comment]
Updated in the version 1.10.10
